upup/pkg/fi/cloudup/scalewaytasks/lb_backend.go

```diff
@@ -163,3 +166,25 @@ func (l *LBBackend) RenderScw(t *scaleway.ScwAPITarget, actual, expected, change
 
 	return nil
 }
+
+type terraformLBBackend struct {
+	LBID            *terraformWriter.Literal `cty:"lb_id"`
+	Name            *string                  `cty:"name"`
+	ForwardProtocol *string                  `cty:"forward_protocol"`
+	ForwardPort     *int32                   `cty:"forward_port"`
+}
+
+func (l *LBBackend) RenderTerraform(t *terraform.TerraformTarget, actual, expected, changes *LBBackend) error {
+	tfName := strings.Replace(fi.ValueOf(expected.LoadBalancer.Name), ".", "-", -1)
```

Comment:
Maybe use `ReplaceAll()` instead?
```suggestion
	tfName := strings.ReplaceAll(fi.ValueOf(expected.LoadBalancer.Name), ".", "-")
```